SAMLPortal/Models/GlobalSettings.cs

```diff
@@ -118,22 +151,22 @@
~			{
~				return _appSettings[key];
~			}
~			catch (KeyNotFoundException ex)
~			{
~				return null;
~			}
 
 		}
 
-		public static int GetInt(string key)
+		public static int? GetInt(string key)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Catch a list of specific exception subtype or use exception filters instead.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)